script/vsts/platforms/cache.yml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  - name: OS
+    displayName: Operating System
+    type: string
+    values:
+      - windows
+      - linux
+      - macos
+
+steps:
+  - task: Cache@2
+    displayName: Cache node_modules
+    inputs:
+      key: 'npm | "$(Agent.OS)" | "$(buildArch)" | package.json, package-lock.json, script/vsts/platforms/{{ parameters.OS }}.yml'
```

Comment:
I think these are missing a `$`:

`${{ parameters.OS }}`

Are we sure this shouldn't be a runtime expression `[parameters.OS]`?

Or maybe `[parameters('OS')]`

Per this documentation: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-parameters#use-parameter

Reply:
Oh! Yes. Did I forget them?! They should have a `$`

Reply:
Okay, that;s for a totally different product "Azure Resource Manager" as opposed to "Azure Pipelines." I'm going to re-read the _Azure Pipelines_ documentation for this.